Content.Server/GameObjects/Components/MachineLinking/SignalTransmitterComponent.cs

```diff
@@ -150,15 +150,17 @@ protected override void Shutdown()
         {
             base.Shutdown();
 
-            foreach (var receiver in _receivers)
+            for (var i = _receivers.Count-1; i <= 0; i++)
```

Comment:
Should this be `for (var i = _receivers.Count - 1; i >= 0; i--)`?

Reply:
yes